### DISCUSSION TITLE
Sharing link-map trees/branches/URLS  and collaborating on trees/branches/sets of URLs

### DISCUSSION THREAD

**Discussion by staalman** [GarinZ/link-map#20] [Ideas]
Feature request 

Support for:
A. Sharing trees or branches.
B. Collaborating on trees or branches.

When using cloud identities (suc as e.g. Google, OpenID, etc.), it will be possible to share and collaborate on (parts of) the tree.
The most straightforward implementation of this may be to use
1. use a shared space on a NAS or on a gdrive to store and maintain link-map URLs 
2. an OpenID or google account as identification, authentication and authorisation instrument, for accessing the file and operations on it, and 
3. use the access rights on the (link-maps stored in the gdrive's) file as a way to support sharing/collaboration of [the] link-map[s]. This allows sharing and collaboration of trees or branches in a controlled manner.  
4. With branches represented as separate files, different user groups can have edit/write rights on different parts of the overall tree.

**Comment by GarinZ** [comment]
Nice idea, I have considered this featured could be added into Link Map some day. I will manage to support it after online storage and sync supported.
